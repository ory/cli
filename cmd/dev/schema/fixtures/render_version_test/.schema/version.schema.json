{
  "$id": "https://github.com/ory/cli/cmd/dev/schema/fixtures/render_version_test/.schema/version.schema.json",
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Test Fixture schema.",
  "type": "object",
  "oneOf": [
    {
      "allOf": [
        {
          "properties": {
            "version": {
              "const": "v0.0.0"
            }
          },
          "required": [
            "version"
          ]
        },
        {
          "$ref": "https://raw.githubusercontent.com/ory/hydra/v0.0.0/.schema/config.schema.json"
        }
      ]
    },
    {
      "allOf": [
        {
          "oneOf": [
            {
              "properties": {
                "version": {
                  "type": "string",
                  "maxLength": 0
                }
              },
              "required": [
                "version"
              ]
            },
            {
              "not": {
                "properties": {
                  "version": {}
                },
                "required": [
                  "version"
                ]
              }
            }
          ]
        },
        {
          "$ref": "#/oneOf/0/allOf/1"
        }
      ]
    }
  ]
}
