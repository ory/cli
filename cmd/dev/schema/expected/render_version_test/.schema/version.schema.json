{
    "$id": "https://github.com/ory/cli/cmd/dev/schema/fixtures/render_version_test/.schema/version.schema.json",
    "$schema": "http://json-schema.org/draft-07/schema#",
    "oneOf": [
        {
            "allOf": [
                {
                    "properties": {
                        "version": {
                            "const": "v1.0.0"
                        }
                    },
                    "required": [
                        "version"
                    ]
                },
                {
                    "$ref": "https://raw.githubusercontent.com/ory/hydra/v1.0.0/.schema/config.schema.json"
                }
            ]
        },
        {
            "allOf": [
                {
                    "properties": {
                        "version": {
                            "const": "v0.0.0"
                        }
                    },
                    "required": [
                        "version"
                    ]
                },
                {
                    "$ref": "https://raw.githubusercontent.com/ory/hydra/v0.0.0/.schema/config.schema.json"
                }
            ]
        },
        {
            "allOf": [
                {
                    "oneOf": [
                        {
                            "properties": {
                                "version": {
                                    "type": "string",
                                    "maxLength": 0
                                }
                            },
                            "required": [
                                "version"
                            ]
                        },
                        {
                            "not": {
                                "properties": {
                                    "version": {}
                                },
                                "required": [
                                    "version"
                                ]
                            }
                        }
                    ]
                },
                {
                    "$ref": "#/oneOf/0/allOf/1"
                }
            ]
        }
    ],
    "title": "Test Fixture schema.",
    "type": "object"
}